src/transformers/models/clip/tokenization_clip.py

```diff
@@ -332,7 +329,12 @@ def _convert_id_to_token(self, index):
     def convert_tokens_to_string(self, tokens):
         """Converts a sequence of tokens (string) in a single string."""
         text = "".join(tokens)
-        text = bytearray([self.byte_decoder[c] for c in text]).decode("utf-8", errors=self.errors).replace("</w>", " ")
+        text = (
+            bytearray([self.byte_decoder[c] for c in text])
+            .decode("utf-8", errors=self.errors)
+            .replace("</w>", " ")
+            .strip()
+        )
```

Comment:
Should be done on several lines since it's reformatted this way.

Reply:
Noted, I made the change in [this commit ](https://github.com/huggingface/transformers/pull/15067/commits/42c4344fff4f762def2caac873a387b4557058c8)